src/main/java/com/logpresso/scanner/Log4j2Scanner.java

```diff
@@ -166,7 +166,9 @@ private void restore(InputStream is, File targetFile) {
 
 			// restore read only attribute
 			if (readonlyFile)
-				targetFile.setReadOnly();
+				if(!targetFile.setReadOnly()) {
+                                   System.out.println("File cannot be set as Read Only");
```

Comment:
User should be notified which file was problem
`System.out.println("Error: File cannot be set as read only - " + targetFile.getAbsolutePath());`